hack/jenkins/test-dockerized.sh

```diff
@@ -44,7 +51,7 @@ export LOG_LEVEL=4
 cd /go/src/k8s.io/kubernetes
 
 ./hack/build-go.sh
-godep go install ./...
+retry godep go install ./...
```

Comment:
has this one actually failed on downloading dependencies? I worry that this will end up retrying a broken build 5 times.